### PR TITLE
OPTC-5

### PR DESCRIPTION
**Pull request by dfernandezramos**

### CODE REVIEW

**Comment by turran** [comment]
The commits can be squashed to not include the deps package
Also, why is an arm related commit on the enums.py?

**Comment by dfernandezramos** [comment]
@turran I was getting errors on cerbero (something like "Architecture has no attribute 'ARM64'") so I defined that arm related commit on the enums.py

**Comment by turran** [comment]
:+1:

File: packages/oneplay-transcoder-deps.package [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        #     if not prefix.endswith('/'):
+        #         prefix += '/'
+        #     files = []
+        #     for x in r:
+        #         p = os.path.realpath(os.path.join(prefix, x))
+        #         files.append(p.replace(prefix, ''))
+        #         config = p+'.config'
+        #         if os.path.exists(config):
+        #             files.append(config.replace(prefix, ''))
+        #     r = list(set(r+files))
```
Comment (ylatuya) [inline packages/oneplay-transcoder-deps.package]: remove commented code

**Comment by ylatuya** [comment]
Since cerbero is the open source part of the repository, the package packages/oneplay-transcoder-deps.package should be moved to fluendo-cerbero

**Comment by ylatuya** [comment]
👍

**Pull request merged**